app/controllers/tasks_controller.rb

```diff
@@ -56,15 +56,15 @@ def set_task
   end
 
   def create_task
-    @create_task ||= ::Tasks::Create.call(task_params: task_params, project: @project)
+    @create_task ||= ::Tasks::Create.call(task_params: task_params, project: @project, task: @task)
```

Comment:
Here `@task` is nil, redundant 
<img width="802" alt="image" src="https://github.com/fs/task_tracker_itis_2023/assets/78980031/21408ae9-bf6e-48c1-a9f8-605cdaa1897f">
```suggestion
    @create_task ||= ::Tasks::Create.call(task_params: task_params, project: @project)
```